etsin_finder/resources/download_resources.py

```diff
@@ -125,6 +127,23 @@ def post(self):
         check_download_permission(cr_id)
 
         scope = args.get('scope')
+
+        projects, status = common_service.get_dataset_projects(cr_id)
+
+        if status != 200:
+            abort(status, message=f"Error occured when Etsin tried to fetch project details from Metax.")
+        if projects is None or len(projects) == 0:
+            abort(500, message=f"Etsin could not find project for dataset using catalog record identifier {cr_id}")
+
+        project = projects[0]
+        directory_details, status = common_service.get_directory_for_project_using_scope(cr_id, project, scope[0])
```

Comment:
Scope can be `None` if the entire dataset is requested. So we need something like:
```suggestion
        directory_details, status = common_service.get_directory_for_project_using_scope(cr_id, project, (scope or ['/'])[0])
```